### PR TITLE
feat: messages 

### PR DESCRIPTION
**Pull request by ghost**
<a href="https://github.com/top-22/forum/actions/workflows/docker-deploy.yaml">
  Docker Deploy workflow status
</a>

[](https://top22-messages.jeschek.eu)

### CODE REVIEW

**Review by philipkbh — APPROVED** [review]
granted :+1:

File: src/pages/room/[roomId]/[threadId].tsx [hunk 1]
```diff
@@ -75,35 +110,41 @@ const Thread: NextPage<ThreadProps> = ({
...
+            </div>
           </div>
           {isJoined && (
             <div className="input-group p-3">
               <input
                 type="text"
                 className="form-control"
                 placeholder="Nachricht"
+                value={messageContent}
+                onChange={handleInputChange}
```
Comment (joshuajeschek) [inline src/pages/room/[roomId]/[threadId].tsx:143]:
```suggestion
                onChange={handleInputChange}
                onKeyDown={(e) => e.key === "Enter" && handleSendMessage()}
```

**Review by joshuajeschek — CHANGES_REQUESTED** [review]
mehehe why not

File: src/pages/room/[roomId]/[threadId].tsx [hunk 2]
```diff
@@ -75,35 +110,41 @@ const Thread: NextPage<ThreadProps> = ({
...
-          <div className="bg-primary m-3 p-3 rounded overflow-auto">
-            {messages.length > 0 ? (
-              messages.map((message) => (
-                <div className="container p-1" key={message.id}>
-                  <div className="row">
-                    <div className="col-auto align-self-center p-0 m-1 w-25">
-                      <div className="d-flex flex-column align-items-start">
-                        <div>{formatTime(new Date(message.createdAt))}</div>
-                        <div>{message.user.name}</div>
+          <div className="bg-primary m-3 p-3 rounded overflow-auto d-flex flex-column-reverse">
```
Comment (joshuajeschek) [inline src/pages/room/[roomId]/[threadId].tsx:113]:
```suggestion
          <div className="bg-primary m-3 p-3 rounded overflow-auto d-flex flex-column-reverse text-white">
```

**Review by joshuajeschek — APPROVED** [review]
ok

**Pull request merged**